examples/ui/feathers.rs

```diff
@@ -0,0 +1,160 @@
+//! This example shows off the various Bevy Feathers widgets.
+
+use bevy::{
+    core_widgets::CoreWidgetsPlugin,
+    ecs::system::SystemId,
+    feathers::{
+        controls::{button, slider, ButtonProps, ButtonVariant, SliderProps},
+        dark::create_dark_theme,
+        theme::{self, corners::RoundedCorners, ThemeBackgroundColor, UiTheme, UseTheme},
+        FeathersPlugin,
+    },
+    input_focus::{tab_navigation::TabGroup, InputDispatchPlugin},
+    prelude::*,
+    ui::InteractionDisabled,
+    winit::WinitSettings,
+};
+
+fn main() {
+    App::new()
+        .add_plugins((
+            DefaultPlugins,
+            CoreWidgetsPlugin,
+            InputDispatchPlugin,
+            FeathersPlugin,
+        ))
+        .insert_resource(UiTheme(create_dark_theme()))
+        // Only run the app when there is user input. This will significantly reduce CPU/GPU use.
+        .insert_resource(WinitSettings::desktop_app())
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(mut commands: Commands) {
+    // System to print a value when the button is clicked.
+    let on_click = commands.register_system(|| {
```

Comment:
I think it would be a slightly nicer showcase if the buttons each printed something custom based on what they are. It would also show a bit more how to have multiple different handlers which seems like a more common case.

Reply:
Done.